packages/@okta/vuepress-theme-default/layouts/Landing.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="Page okta-landing">
+    <TopNavigation />
+    <Breadcrumb />
+    <section class="PageContent NoPaddingTop">
+      <!-- START Page Content -->
+      <div class="PageContent-main">
+        <div class="landing-wrap">
+          <div class="landing-page-content">
+            <Sidebar :oldNavigation=false />
+            <div class="landing-content">
+              <Content />
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- END Page Content -->
+    </section>
+    <Footer/>
+  </div>
+</template>
+
+<script>
+export default {
+  components: {
+    TopNavigation: () => import('../components/TopNavigation.vue'),
+    Footer: () => import('../components/Footer.vue'),
+    Sidebar: () => import('../components/Sidebar.vue')
+  }
+}
+
+</script>
+
+<style lang="scss">
+  @import '../assets/css/okta';
+  @import '~prismjs/themes/prism-solarizedlight.css';
+  .icon.outbound {
```

Comment:
We have this in a few places now, IIRC - should we move it to a common single location?

Reply:
Once we re-do the main layout of the site, ill migrate this out